app/Exceptions/Handlers/AccessDBDenied.php

```diff
@@ -18,17 +18,17 @@ class AccessDBDenied
 	 *
 	 * @return bool
 	 */
-	public function check($request, Throwable $exception)
+	public function check(Request $request, Throwable $exception)
 	{
 		// encryption key does not exist, we need to run the installation
 		return $exception instanceof QueryException && (strpos($exception->getMessage(), 'Access denied') !== false);
 	}
 
 	/**
-	 * @return Response
+	 * @return Request
```

Comment:
Are you sure it is Request? 

`ToInstall::go()` has for code:
```php
		return redirect(route('install-welcome'), 307, [
			'Cache-Control' => 'no-cache, must-revalidate',
		]);
```
which has return type: RedirectResponse :thinking:  
https://laravel.com/docs/8.x/redirects

Unless I'm wrong.

Reply:
My fault.